### ISSUE TITLE
Boxen installation fails

### ISSUE THREAD

**Issue by aripalo** [boxen/our-boxen#385]
I just can't get boxen working on our dev machines.

I think it has something to do with the username, since weirdly boxen does a lot of this kind of stuff: 
```Notice: /Stage[main]/Boxen::Config/File[/opt/boxen]/owner: owner changed '400706' to '400706' ``` 
That seems quite weird. And to clarify '400706' really is my username due to a company policy.

Then also I get a lot of ```Dependency File[/opt/boxen/bin/boxen-git-credential] has failures: true``` errors, even though I have given correct Github login information at script/boxen start.

If anybody would happen to have any ideas how to I might get boxen working or at least where to look I'd appreciate it much since I'm really out of ideas here!

Here's my boxen.log output: https://gist.github.com/aripalo/6269191

**Comment by seanknox** [comment]
Closing this since been a while since it was originally reported. If it's still happening feel free to reopen.

**Comment by aripalo** [comment]
Yeah, can be closed... can't actually even remember if we got that fixed or not. 

We actually moved away from Boxen as it turned out to be to too much of work for our team to manage. Nowadays our ICT guys provide us easy, managed software installation via [Munki](https://github.com/munki/munki) and all our development is done in Vagrant+Ansible controlled virtual machines. So there's not really that much need for Boxen's features.
